libtac/lib/attrib.c

```diff
@@ -20,42 +20,67 @@
  * See `CHANGES' file for revision history.
  */
 
+#include <stdbool.h>
+
```

Comment:
`stdbool.h` seems to be C99 specific, don't we need to check for in `configure.in`?

https://en.wikibooks.org/wiki/C_Programming/stdbool.h

Reply:
Sure, there was not particular reason to use bool though, so it seems simpler to just use int.

I'll update the PR.